docs/Security-Compliance/Security-Controls/AC/AC-05-Separation-of-duties.md

```diff
@@ -0,0 +1,62 @@
+
+***Access to the Django admin interface***  
+
+Access directly to the Django interface occurs through a combination of flags and assigned permissions based on responsibilities.  
+
+- **active flag:** This flag, on creation of a new user, is set to true. Users that are not flagged as active are not able to login into the system and considered inactive. This flag could be changed in any of the following circumstances (and, others):
+
+  - deactivation of a user
+  - reactivation of a user  
+  
+- **superuser flag:** On creation of a new user, this flag is set to false. Creation of a superuser must be done explicitly. If a user is flagged and logged in as a superuser, the user has permissions to create, read, update and delete data in the Django admin, which includes model records and other users.  
+
+- **staff flag:** This flag, on creation of a new user, is set to false. If a user is flagged and logged in as staff, the user can log into the Django admin. If a staff flag is attached to a user, this differentiates the user as an administrator of the system versus a public user. Active, non-superuser staff members are granted administrative access through assigned permissions.  
+
+## Access to the TDP  
+
+ #### Overview
+- **Login:** Login to the system GET request is sent to ***v1/login/oidc endpoint***. Not authorization protected. Users authenticate with Login.gov via the system’s backend service. Upon successful authentication, the backend service will create httpOnly cookie and stored by the clients browser for authenticating future requests to the API. The secured portion of this authorization is due to the httpOnly cookie being inaccessible to the client’s local browser. The user is then redirected to the frontend landing page.
+
+- **On-going Authorization:** Authorization checks are carried out with GET requests to ***v1/auth_check*** endpoint where the JSON response that includes profile information including “authenticated”: True. This also refreshes the user session. Authorization is protected.
+
+- **Logout:** Logout of the system is through a GET request to ***v1/logout/oidc***. Not Authorization protected. This will destroy the current user session while also logging out of Login.gov, after which, the user is redirected to the landing page.
+
+- **Updating Profile:** In order for users to update their profile, a PATCH request is sent to ***v1/set_profile*** endpoint. This action can be performed through the TDP interface by the user or an administrator.  
+- The following endpoints, which are protected, are accessed to view data if the user is authenticated:  
+
+  - GET ***/v1/stts/*** Returns a JSON response with a list of all States, Tribes and Territories in the system
+  - GET ***/v1/stts/alpha*** Returns a JSON response as above but sorted by the “name” (of the entity).
+  - GET ***/v1/stts/by_region*** Returns a JSON response sodrted by “stts” (the top level region)
```

Comment:
This is a lot of technical detail — which isn't a bad thing at all! But we may want to check with OCIO to understand if this level of technical detail is required for this particular control documentation.

Reply:
Would the following be sufficient, or is this too little detail?

**Part c.**   
Access to the TDP occurs in two different ways: through the Django admin interface and through the TDP application interface.  

 ***Access to the Django admin interface***  

 Access directly to the Django interface occurs through a combination of flags and assigned permissions based on responsibilities.  

## Access to the TDP  

Users authenticate with Login.gov via the system’s backend service. Upon successful authentication the user is then redirected to the frontend landing page.

Reply:
I can't say for certain -- again, we'll want to check with OCIO -- but it might be sufficient! I'd be curious to find out